app/models/user.rb

```diff
@@ -9,4 +9,11 @@ class User < ApplicationRecord
   has_many :following, through: :active_relationships, source: :followed
   has_many :passive_relationships, class_name: 'Relationship', foreign_key: 'followed_id', dependent: :destroy, inverse_of: :followed
   has_many :followers, through: :passive_relationships, source: :follower
+
+  # 名前系：100文字以内、説明系：256文字以内で一旦設定
+  validates :name, presence: true, length: { maximum: 100 }
+  validates :email, presence: true, uniqueness: { case_sensitive: false }, format: { with: URI::MailTo::EMAIL_REGEXP, message: 'must be a valid email format' }
```

Comment:
他のバリデーションエラーは日本語で返ってくるので、こちらも日本語でお願いします！
```suggestion
  validates :email, presence: true, uniqueness: { case_sensitive: false }, format: { with: URI::MailTo::EMAIL_REGEXP, message: 'のフォーマットが正しくありません }
```

Reply:
ありがとうございます！